packages/api/src/api-key/controllers.ts

```diff
@@ -0,0 +1,70 @@
+import { Request } from 'express';
+import { Types } from 'mongoose';
+import { BadRequestError, NotFoundError } from '@/error/errors';
+import {
+  QueryInputParsedQs,
+  TypedRequestBody,
+  TypedRequestQuery,
+  TypedResponse,
+} from '@/shared/types';
+import { EventLogTypeKey } from '@/eventlog/types';
+import { logEvent } from '@/eventlog/utils';
+import { ApiKeyModel } from './entities';
+// import { userListTransformer, userTransformer } from './transformers';
+import { ApiKeyDocument, ApiKeyDto, ApiKeyAccess } from './types';
+import { findUser } from './utils/entity';
+
+/**
+ * Fetch all Users with their associated UserAccounts
+ *
+ * @param {TypedRequestQuery<QueryInputParsedQs>} req
+ * @param {TypedResponse<PaginatedResponseBody<ApiKeyDto[]>>} res
+ * @returns {Promise<void>}
+ */
+export const all = async (
+  req: TypedRequestQuery<QueryInputParsedQs>,
+  res: TypedResponse<ApiKeyDto[]>
+): Promise<void> => {
+  const apiKeys = await ApiKeyModel.paginate({
```

Comment:
We don't need to do pagination for api keys as we don't expect there to be a large number of keys